src/Parser/Paladin/Retribution/Modules/PaladinCore/Judgment.js

```diff
@@ -58,16 +58,27 @@ class Judgment extends Analyzer {
 		}
 	}
 
-	suggestions(when) {
+	get suggestionThresholds() {
 		const unbuffedJudgmentPercentage = this.spenderOutsideJudgment / this.totalSpender;
-		when(unbuffedJudgmentPercentage).isGreaterThan(0.05)
+		return {
+			actual: unbuffedJudgmentPercentage,
+			isGreaterThan: {
+				minor: 0.05,
+				average: 0.1,
+				major: 0.15,
+			},
+			style: 'percentage',
+		};
+	}
+
+	suggestions(when) {
+		when(this.suggestionThresholds)
 			.addSuggestion((suggest,actual,recommended) => {
 				return suggest(<span>You're spending Holy Power outisde of the <SpellLink id={SPELLS.JUDGMENT_CAST.id} /> debuff. It is optimal to only spend Holy Power while the enemy is debuffed with <SpellLink id={SPELLS.JUDGMENT_CAST.id} />.</span>)
 					.icon(SPELLS.JUDGMENT_DEBUFF.icon)
 					.actual(`${formatNumber(this.spenderOutsideJudgment)} Holy Power spenders used outside of Judgment (${formatPercentage(actual)}%).`)
-					.recommended(`<${formatPercentage(recommended)}% is recommended`)
-					.regular(recommended + 0.05).major(recommended + 0.1);
-			});
+					.recommended(`<${formatPercentage(recommended)}% is recommended`);
+				});
 	}
 
```

Comment:
some styling with the indentations here
It's probably safe to move the .addSuggestion() up next to when() on all of them